src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -344,6 +354,16 @@ private function parseDefinition(\DOMElement $service, $file, array $defaults)
             $definition->addAutowiringType($type->textContent);
         }
 
+        $bindings = $this->getArgumentsAsPhp($service, 'bind', $file);
+        if (isset($defaults['bind'])) {
+            // deep clone, to avoid multiple process of the same instance in the
+            // passes
```

Comment:
comment should be on one line
`// deep clone, to avoid processing the same definitions several times in  compiler passes`